code/modules/jobs/job_types/security_officer.dm

```diff
@@ -281,6 +281,13 @@ GLOBAL_LIST_INIT(available_depts, list(SEC_DEPT_ENGINEERING, SEC_DEPT_MEDICAL, S
 	id = /obj/item/card/id/syndicate_command/crew_id
 	backpack_contents = list(/obj/item/storage/box/survival/syndie=1)
 
+/datum/outfit/job/security/pirate/pilot
```

Comment:
Why is this a subtype of security officer? "first mate" positions are typically a subtype of HoP and treated as officer ranks. You might want to use the existing pirate HoP outfit for this slot.